### PR TITLE
Update readme

### LINKED ISSUE #62: Document CloudRunJob better in README
It's currently:
- missing install cloud run
- how to register CloudRunJob block.

### PR DESCRIPTION
**Pull request by ahuang11**
Improves visibility on Cloud Run.

Closes #62

### Checklist

- [ ] This pull request references any related issue by including "Closes #<ISSUE_NUMBER>"
	- If no issue exists and your change is not a small fix, please [create an issue](https://github.com/PrefectHQ/prefect-gcp/issues/new/choose) first.
- [x] This pull request includes tests or only affects documentation.
- [ ] Summarized PR's changes in [CHANGELOG.md](https://github.com/PrefectHQ/prefect-gcp/blob/main/CHANGELOG.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,29 +54,50 @@ To use Secret Manager:
...
+
+```python
+from prefect import flow
+from prefect_gcp import GcpCredentials
+from prefect_gcp.cloud_run import CloudRunJob
+
+@flow
+def cloud_run_job_flow():
+    cloud_run_job = CloudRunJob(
+        image="us-docker.pkg.dev/cloudrun/container/job:latest",
```
Comment (desertaxle) [inline README.md:93]: I think that we can use the prefect Docker images as the example image.

File: README.md [hunk 2]
```diff
@@ -54,29 +54,50 @@ To use Secret Manager:
...
     gcp_credentials = GcpCredentials(
         service_account_file="/path/to/service/account/keyfile.json")
     contents = cloud_storage_download_blob_as_bytes("bucket", "blob", gcp_credentials)
     return contents
 
-example_cloud_storage_download_blob_flow()
+cloud_storage_download_blob_flow()
+```
+
+### Run command on Cloud Run
```
Comment (desertaxle) [inline README.md:83]: This is a good example to have, but I think we need an example specifically for how we can use the `CloudRunJob` block in deployments. I think we could have a simple example here and link back to the core docs, but I'm open to other approaches.

File: README.md [hunk 3]
```diff
@@ -32,7 +32,7 @@ These tasks are designed to work with Prefect 2.0. For more information about ho
 
 ### Installation
 
-Install `prefect-gcp` with `pip`:
+To use `prefect-gcp` and Cloud Run:
```
Comment (desertaxle) [inline README.md:35]: I don't think that Cloud Run needs to be mentioned in the installation instructions. But I do think that we should include a section above installation that lists some of the thinks that you can do with this collection as a hook for users.
```suggestion
Install `prefect-gcp` with `pip`:
```

**Review by desertaxle — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (ahuang11) [reply README.md:35]: I think it's clearer if we mention this also installs Cloud Run since all the other modules get mentioned.

**Comment by ahuang11** [comment]
Docker image needs updating.

**Comment by ahuang11** [comment]
I couldn't get/find a prefect image on gcr.

**Pull request merged**